### PR TITLE
Fix of selinux directories

### LINKED ISSUE #598: [BUG] line 16: fg: no job control when installing RPM on Fedora 35
#### What happened:

When installing microshift RPM on Fedora 35 (x86_64) it gives this error:

````
/var/tmp/rpm-tmp.zpB674: line 16: fg: no job control
````

The package seems to be installed correctly. It might be some problem in the rpm SPEC, because the /var/hpvolumes directory does not get the correct SELinux context after the installation:

````
$ ls -ldZ /var/hpvolumes/
drwxr-xr-x. 1 root root system_u:object_r:var_t:s0 0 Feb  3 09:49 /var/hpvolumes/
````

This is the complete dnf message:

````
$ sudo dnf install -y microshift

Dependencies resolved.
======================================================================================================================================================
 Package                   Architecture  Version                               Repository                                                        Size
======================================================================================================================================================
Installing:
 microshift                x86_64        4.8.0-2022_02_02_194009.fc35          copr:copr.fedorainfracloud.org:group_redhat-et:microshift         34 M
Installing dependencies:
 microshift-selinux        noarch        4.8.0-2022_02_02_194009.fc35          copr:copr.fedorainfracloud.org:group_redhat-et:microshift         19 k

Transaction Summary
======================================================================================================================================================
Install  2 Packages

Total download size: 34 M
Installed size: 154 M
Downloading Packages:
(1/2): microshift-selinux-4.8.0-2022_02_02_194009.fc35.noarch.rpm                                                     337 kB/s |  19 kB     00:00
(2/2): microshift-4.8.0-2022_02_02_194009.fc35.x86_64.rpm                                                              27 MB/s |  34 MB     00:01
------------------------------------------------------------------------------------------------------------------------------------------------------
Total                                                                                                                  27 MB/s |  34 MB     00:01
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                                                              1/1
  Installing       : microshift-selinux-4.8.0-2022_02_02_194009.fc35.noarch                                                                       1/2
  Running scriptlet: microshift-selinux-4.8.0-2022_02_02_194009.fc35.noarch                                                                       1/2
/var/tmp/rpm-tmp.zpB674: line 16: fg: no job control
warning: %post(microshift-selinux-4.8.0-2022_02_02_194009.fc35.noarch) scriptlet failed, exit status 1

Error in POSTIN scriptlet in rpm package microshift-selinux
  Installing       : microshift-4.8.0-2022_02_02_194009.fc35.x86_64                                                                               2/2
  Running scriptlet: microshift-4.8.0-2022_02_02_194009.fc35.x86_64                                                                               2/2
  Running scriptlet: microshift-selinux-4.8.0-2022_02_02_194009.fc35.noarch                                                                       2/2
  Running scriptlet: microshift-4.8.0-2022_02_02_194009.fc35.x86_64                                                                               2/2
  Verifying        : microshift-4.8.0-2022_02_02_194009.fc35.x86_64                                                                               1/2
  Verifying        : microshift-selinux-4.8.0-2022_02_02_194009.fc35.noarch                                                                       2/2

Installed:
  microshift-4.8.0-2022_02_02_194009.fc35.x86_64                        microshift-selinux-4.8.0-2022_02_02_194009.fc35.noarch

Complete!
````

#### What you expected to happen:

The installation completes without a warning.

#### How to reproduce it (as minimally and precisely as possible):

1. Install COPR repo
2. Install microshift RPM package
3. It gives a warning "line 16: fg: no job control"

#### Environment:
- Microshift version (use `microshift version`):
````
MicroShift Version: 4.8.0-0.microshift-2022-02-02-194009
Base OKD Version: 4.8.0-0.okd-2021-10-10-030117
````

- Hardware configuration:
VM with 8 GB of RAM and 2 vCPUs.

- OS (e.g: `cat /etc/os-release`):
````
cat /etc/os-release
NAME="Fedora Linux"
VERSION="35 (Cloud Edition)"
ID=fedora
VERSION_ID=35
VERSION_CODENAME=""
PLATFORM_ID="platform:f35"
PRETTY_NAME="Fedora Linux 35 (Cloud Edition)"
ANSI_COLOR="0;38;2;60;110;180"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:35"
HOME_URL="https://fedoraproject.org/"
DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/f35/system-administrators-guide/"
SUPPORT_URL="https://ask.fedoraproject.org/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=35
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=35
PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
VARIANT="Cloud Edition"
VARIANT_ID=cloud
````

- Kernel (e.g. `uname -a`):
````
Linux microshift 5.15.18-200.fc35.x86_64 #1 SMP Sat Jan 29 13:54:17 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
````

### PR DESCRIPTION
**Pull request by cooktheryan**
Modify the way we are creating and labeling directories

Closes #598

### CODE REVIEW

**Comment by cooktheryan** [comment]
```
[root@ip-172-31-87-50 ~]# dnf install microshift
Updating Subscription Management repositories.
Copr repo for microshift owned by cooktheryan                                         38 kB/s | 2.6 kB     00:00    
Dependencies resolved.
=====================================================================================================================
 Package  Arch   Version                                 Repository                                             Size
=====================================================================================================================
Installing:
 microshift
          x86_64 4.8.0-2022_02_02_194009_3_gc5c2c9d6.el8 copr:copr.fedorainfracloud.org:cooktheryan:microshift  36 M
Installing dependencies:
 microshift-selinux
          noarch 4.8.0-2022_02_02_194009_3_gc5c2c9d6.el8 copr:copr.fedorainfracloud.org:cooktheryan:microshift  20 k

Transaction Summary
=====================================================================================================================
Install  2 Packages

Total download size: 36 M
Installed size: 154 M
Is this ok [y/N]: y
Downloading Packages:
(1/2): microshift-selinux-4.8.0-2022_02_02_194009_3_gc5c2c9d6.el8.noarch.rpm         334 kB/s |  20 kB     00:00    
(2/2): microshift-4.8.0-2022_02_02_194009_3_gc5c2c9d6.el8.x86_64.rpm                  19 MB/s |  36 MB     00:01    
---------------------------------------------------------------------------------------------------------------------
Total                                                                                 19 MB/s |  36 MB     00:01     
Copr repo for microshift owned by cooktheryan                                         49 kB/s | 1.0 kB     00:00    
Importing GPG key 0x26DF568C:
 Userid     : "cooktheryan_microshift (None) <cooktheryan#microshift@copr.fedorahosted.org>"
 Fingerprint: 6075 9B78 E731 4E58 7E4F E878 AA94 B3E7 26DF 568C
 From       : https://download.copr.fedorainfracloud.org/results/cooktheryan/microshift/pubkey.gpg
Is this ok [y/N]: y
Key imported successfully
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                             1/1 
  Installing       : microshift-selinux-4.8.0-2022_02_02_194009_3_gc5c2c9d6.el8.noarch                           1/2 
  Running scriptlet: microshift-selinux-4.8.0-2022_02_02_194009_3_gc5c2c9d6.el8.noarch                           1/2 
  Installing       : microshift-4.8.0-2022_02_02_194009_3_gc5c2c9d6.el8.x86_64                                   2/2 
  Running scriptlet: microshift-4.8.0-2022_02_02_194009_3_gc5c2c9d6.el8.x86_64                                   2/2 
  Running scriptlet: microshift-selinux-4.8.0-2022_02_02_194009_3_gc5c2c9d6.el8.noarch                           2/2 
  Running scriptlet: microshift-4.8.0-2022_02_02_194009_3_gc5c2c9d6.el8.x86_64                                   2/2 
  Verifying        : microshift-4.8.0-2022_02_02_194009_3_gc5c2c9d6.el8.x86_64                                   1/2 
  Verifying        : microshift-selinux-4.8.0-2022_02_02_194009_3_gc5c2c9d6.el8.noarch                           2/2 
Installed products updated.

Installed:
  microshift-4.8.0-2022_02_02_194009_3_gc5c2c9d6.el8.x86_64                                                          
  microshift-selinux-4.8.0-2022_02_02_194009_3_gc5c2c9d6.el8.noarch                                                  

Complete!

[root@ip-172-31-87-50 ~]# ls -lZ /var/
drwxr-xr-x.  2 root root unconfined_u:object_r:container_file_t:s0    6 Feb  7 16:10 hpvolumes
```

File: packaging/rpm/microshift.spec [hunk 1]
```diff
@@ -175,9 +185,7 @@ fi
 %post selinux
 
 %selinux_modules_install -s %{selinuxtype} %{_datadir}/selinux/packages/%{selinuxtype}/microshift.pp.bz2
-if /usr/sbin/selinuxenabled ; then
-    %microshift_relabel_files
-fi;
+%microshift_relabel_files
```
Comment (mangelajo) [inline packaging/rpm/microshift.spec:188]: Hmm, didn't we have that macro before? where is it gone?

Weren't we supposed to only relabel if selinux is enabled?

**Comment by cooktheryan** [comment]
@mangelajo sorry about that. Patched.

```
[ec2-user@ip-172-31-91-72 ~]$ sudo dnf -y install microshift
Updating Subscription Management repositories.
Copr repo for microshift owned by cooktheryan                                         50 kB/s | 2.6 kB     00:00    
Dependencies resolved.
=====================================================================================================================
 Package  Arch   Version                                 Repository                                             Size
=====================================================================================================================
Installing:
 microshift
          x86_64 4.8.0-2022_02_02_194009_4_g76349837.el8 copr:copr.fedorainfracloud.org:cooktheryan:microshift  36 M
Installing dependencies:
 microshift-selinux
          noarch 4.8.0-2022_02_02_194009_4_g76349837.el8 copr:copr.fedorainfracloud.org:cooktheryan:microshift  20 k

Transaction Summary
=====================================================================================================================
Install  2 Packages

Total download size: 36 M
Installed size: 154 M
Downloading Packages:
(1/2): microshift-selinux-4.8.0-2022_02_02_194009_4_g76349837.el8.noarch.rpm         662 kB/s |  20 kB     00:00    
(2/2): microshift-4.8.0-2022_02_02_194009_4_g76349837.el8.x86_64.rpm                  22 MB/s |  36 MB     00:01    
---------------------------------------------------------------------------------------------------------------------
Total                                                                                 22 MB/s |  36 MB     00:01     
Copr repo for microshift owned by cooktheryan                                         44 kB/s | 1.0 kB     00:00    
Importing GPG key 0x26DF568C:
 Userid     : "cooktheryan_microshift (None) <cooktheryan#microshift@copr.fedorahosted.org>"
 Fingerprint: 6075 9B78 E731 4E58 7E4F E878 AA94 B3E7 26DF 568C
 From       : https://download.copr.fedorainfracloud.org/results/cooktheryan/microshift/pubkey.gpg
Key imported successfully
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                             1/1 
  Installing       : microshift-selinux-4.8.0-2022_02_02_194009_4_g76349837.el8.noarch                           1/2 
  Running scriptlet: microshift-selinux-4.8.0-2022_02_02_194009_4_g76349837.el8.noarch                           1/2 
  Installing       : microshift-4.8.0-2022_02_02_194009_4_g76349837.el8.x86_64                                   2/2 
  Running scriptlet: microshift-4.8.0-2022_02_02_194009_4_g76349837.el8.x86_64                                   2/2 
  Running scriptlet: microshift-selinux-4.8.0-2022_02_02_194009_4_g76349837.el8.noarch                           2/2 
  Running scriptlet: microshift-4.8.0-2022_02_02_194009_4_g76349837.el8.x86_64                                   2/2 
  Verifying        : microshift-4.8.0-2022_02_02_194009_4_g76349837.el8.x86_64                                   1/2 
  Verifying        : microshift-selinux-4.8.0-2022_02_02_194009_4_g76349837.el8.noarch                           2/2 
Installed products updated.

Installed:
  microshift-4.8.0-2022_02_02_194009_4_g76349837.el8.x86_64                                                          
  microshift-selinux-4.8.0-2022_02_02_194009_4_g76349837.el8.noarch                                                  

Complete!
[ec2-user@ip-172-31-91-72 ~]$ ls -lZ /var/
total 8
drwxr-xr-x.  2 root root system_u:object_r:var_t:s0                   6 Jun 21  2021 adm
drwxr-xr-x. 10 root root system_u:object_r:var_t:s0                 117 Feb  7 16:53 cache
drwxr-xr-x.  2 root root system_u:object_r:kdump_crash_t:s0           6 Nov  2 07:50 crash
drwxr-xr-x.  3 root root system_u:object_r:system_db_t:s0            18 Jun 21  2021 db
drwxr-xr-x.  3 root root system_u:object_r:var_t:s0                  18 Jun 21  2021 empty
drwxr-xr-x.  2 root root system_u:object_r:public_content_t:s0        6 Jun 21  2021 ftp
drwxr-xr-x.  2 root root system_u:object_r:games_data_t:s0            6 Jun 21  2021 games
drwxr-xr-x.  2 root root system_u:object_r:var_t:s0                   6 Jun 21  2021 gopher
drwxr-xr-x.  2 root root unconfined_u:object_r:container_file_t:s0    6 Feb  7 17:05 hpvolumes
drwxr-xr-x.  3 root root system_u:object_r:var_t:s0                  18 Aug 25 15:20 kerberos
drwxr-xr-x. 34 root root system_u:object_r:var_lib_t:s0            4096 Feb  7 17:04 lib
drwxr-xr-x.  2 root root system_u:object_r:var_t:s0                   6 Jun 21  2021 local
lrwxrwxrwx.  1 root root system_u:object_r:var_lock_t:s0             11 May  4  2021 lock -> ../run/lock
drwxr-xr-x. 11 root root system_u:object_r:var_log_t:s0            4096 Feb  7 17:03 log
lrwxrwxrwx.  1 root root system_u:object_r:mail_spool_t:s0           10 Jun 21  2021 mail -> spool/mail
drwxr-xr-x.  2 root root system_u:object_r:var_t:s0                   6 Jun 21  2021 nis
drwxr-xr-x.  2 root root system_u:object_r:var_t:s0                   6 Jun 21  2021 opt
drwxr-xr-x.  2 root root system_u:object_r:var_t:s0                   6 Jun 21  2021 preserve
lrwxrwxrwx.  1 root root system_u:object_r:var_run_t:s0               6 May  4  2021 run -> ../run
drwxr-xr-x.  7 root root system_u:object_r:var_spool_t:s0            68 Jun 21  2021 spool
drwxrwxrwt.  4 root root system_u:object_r:tmp_t:s0                 103 Feb  7 17:05 tmp
drwxr-xr-x.  2 root root system_u:object_r:var_yp_t:s0                6 Jun 21  2021 yp
[ec2-user@ip-172-31-91-72 ~]$ 
```

**Pull request merged**